packages/portalnetwork/src/subprotocols/history/contentManager.ts

```diff
@@ -41,8 +38,9 @@ export class ContentManager {
 
     switch (contentType) {
       case HistoryNetworkContentTypes.BlockHeader: {
+        const headerWithProof = BlockHeaderWithProof.deserialize(value)
```

Comment:
Should the `value` passed into `addContentToHistory` move from being an RLP serialized header or block body or receipt to being an SSZ serialized object?  Looks like a variety of tests around this are failing because this function is now expecting an SSZ serialized value.  I don't think there's a right or wrong answer but not sure which way makes more sense.  We originally built this method as a convenience method to handle blocks and headers imported via RPC but it gets murky once you add in epoch accumulators.  I'm not sure I have a clear view on this but if we want outside users to consume our code, I kinda think we should keep the input (at least at the protocol level) as an RLP encoded header or block.  It might make sense to rewrite the `contentManager` method to expect SSZ serialized portal network objects and leave `addContentToHistory` as a convenience method exposes by the provider or downstream consumer.

Reply:
>  It might make sense to rewrite the contentManager method to expect SSZ serialized portal network objects and leave addContentToHistory as a convenience method exposes by the provider or downstream consumer.

I agree with this.  I don't think it makes sense to separate the header and the proof once validated.    

'addContentToHistory' can be a convenience method, and the actual DB storage and retrieval can be simpler. :thumbsup